core/src/vs/conv.rs

```diff
@@ -75,11 +75,14 @@ pub fn try_u128_to_versionstamp(v: u128) -> Result<[u8; 10], Error> {
 	Ok(buf)
 }
 
+/// Take the most significant, time-based bytes and ignores the last 2 bytes
+#[doc(hidden)]
```

Comment:
Considering this was previously in docs.rs, and with the thought that it now may change in the future as it's not in docs.rs, is anyone possibly relying on this method?

Reply:
Nobody should be relying on this - I don't think this is exposed from the lib.rs chain